validator/src/command.rs

```diff
@@ -0,0 +1,254 @@
+use std::fmt::{Debug, Display};
+use std::fs;
+use std::str::FromStr;
+
+use anchor_client::solana_sdk::signature::{
+    read_keypair_file, Keypair, Signer,
+};
+use clap::{arg, command, Args, Parser, Subcommand};
+use dialoguer::theme::ColorfulTheme;
+use dialoguer::Input;
+use log::LevelFilter;
+use serde::de::Error as SerdeError;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use serde_bytes::{ByteBuf as SerdeByteBuf, Bytes as SerdeBytes};
+
+use crate::utils::{config_file, setup_logging};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Config {
+    pub rpc_url: String,
+    pub ws_url: String,
+    pub program_id: String,
+    pub genesis_hash: String,
+    pub keypair: InnerKeypair,
+    pub log_level: String,
+}
+
+pub struct InnerKeypair(Keypair);
```

Comment:
```suggestion
#[derive(derive_more::From, derive_more::Into)]
pub struct InnerKeypair(Keypair);
```

and then you don’t have to implement From and Into manually.

Reply:
nice...